### ISSUE TITLE
[1.12.2] MetalChests Dupe glitch

### ISSUE THREAD

**Issue by bigenergy** [T145/metalchests#109]
## Problem Details

1. We put an induction furnace, we connect energy to it.
2. We take 2 improvements from IC2: one for automatic extraction, the other for automatic placement. We adjust the improvements on different sides and insert them into the oven.
3. On the same side we put copper chests from MetalChests.
4. We put dust in a copper chest for car parking (possibly - any subject for overcooking, checked for iridium and uranium dust)
5. The stove begins to hammer in the dust, which we put ad infinitum, but it does not disappear from the chest. Plus, for each remelted ingot in the chest for the result, they get much more (there was something like 10,46,64 for each iteration of the stove)

Video: https://www.youtube.com/watch?v=TD385ckX6Sc&feature=youtu.be
This glitch only works with MetalChests chests.

## Environment

-   OS: Ubuntu 19.04
-   Full Forge version: 1.12.2-14.23.5.2838
-   Mod version: MetalChests-v6.1.0 + TBone-v1.6.8

**Comment by T145** [comment]
Ok, but does this happen w/ mods that have something w/ a similar machine, like Thermal Expansion? If not then it's an issue w/ the Induction Furnace.

**Comment by Lewui** [comment]
https://www.youtube.com/watch?v=lDfg7ZAcEFI&feature=youtu.be

**Comment by Lewui** [comment]
https://www.youtube.com/watch?v=lDfg7ZAcEFI&feature=youtu.be

**Comment by T145** [comment]
Yes, but is it a problem w/ other mods? If not, then it's just a problem related specifically to IC2.

**Comment by bigenergy** [comment]
This only works with chests from MetalChests.

**Comment by bigenergy** [comment]
@T145 any ideas?

**Comment by bigenergy** [comment]
This also works with the resource exchanger IC2. Only chests are vulnerable form metalchests.

**Comment by T145** [comment]
@bigenergy No, not really. Since it only seems to be something that happens w/ IC2, then it must be w/ something in how they handle their capabilities. Knowing how convoluted some of the code can get in mods like that, Railcraft & GregTech, I'd just assume they're doing something weird. This wouldn't be the first time item dupe bugs have occurred w/ IC2.

**Comment by bigenergy** [comment]
Sorry, I had to go back to ironchest. Since it does not work with their chests.

**Comment by T145** [comment]
I'll try to look into this interaction in more detail after porting MC over to 1.15. Things likely changed in Forge and IC2 that caused this problem.
